root/area/AreaPlaces.js

```diff
@@ -36,8 +37,19 @@ const AreaPlaces = ({
 
     {places?.length ? (
       <>
-        <div id="largemap" />
-        {manifest.js('area/places-map.js', {'data-args': mapDataArgs})}
+        {DBDefs.MAPBOX_ACCESS_TOKEN ? (
+          <>
+            <div id="largemap" />
+            {manifest.js('area/places-map.js', {'data-args': mapDataArgs})}
+          </>
+        ) : (
+          <p>
+            {l(
+              `A map cannot be shown because maps service access token
+               is not set for this server.`,
```

Comment:
Nitpick, maybe `A map cannot be shown because no maps service access token has been set for this server.` reads better?

Reply:
Changed in https://github.com/metabrainz/musicbrainz-server/pull/1334/commits/7219ad43fdc0f3347099cdf972fc2d27fa167745, thanks.